<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupSimulation</class>
 <widget class="QWidget" name="SetupSimulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>251</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string>Parameter Explanation</string>
   </property>
   <widget class="QTextBrowser" name="explanationText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>481</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFormatting">
     <set>QTextEdit::AutoBulletList</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>481</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string>Parameters:</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="range">
       <property name="text">
        <string>Hunter Travel Range (km):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QToolButton" name="huntHelp">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="carry">
       <property name="text">
        <string>Carrying Capacity:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="carryValue"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="rangeValue"/>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="carryHelp">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="durationValue"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="hphy">
       <property name="text">
        <string>Hunts per Hunter per Year:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="duration">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Simulation Duration (years):</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QToolButton" name="killHelp">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="rangeHelp">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="huntValue"/>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="durationHelp">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="killValue"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="killRate">
       <property name="text">
        <string>Hunter Kill Probability:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="height">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="width">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="heightValue"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="widthValue"/>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="dimensions">
       <property name="text">
        <string>Map Dimensions (km):</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QToolButton" name="dimensionsHelp">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="geolocation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Geolocation of upper left corner: (optional)</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QToolButton" name="geoHelp">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="latitude">
         <property name="text">
          <string>Latitude:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="longitude">
         <property name="text">
          <string>Longitude:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="latValue"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="longValue"/>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="growthLabel">
       <property name="text">
        <string>Animal Growth Rate:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="growthValue"/>
     </item>
     <item row="6" column="2">
      <widget class="QToolButton" name="growthHelp">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="submit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>560</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QLabel" name="Header">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter Simulation Parameters:</string>
   </property>
  </widget>
  <widget class="QLabel" name="explanation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Please enter values for the following parameters to setup the simulation. The button to the right of each option provides an explanation of the parameter.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="advancedOptions">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>560</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Advanced Options</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>explanationText</tabstop>
  <tabstop>durationValue</tabstop>
  <tabstop>carryValue</tabstop>
  <tabstop>rangeValue</tabstop>
  <tabstop>huntValue</tabstop>
  <tabstop>killValue</tabstop>
  <tabstop>growthValue</tabstop>
  <tabstop>heightValue</tabstop>
  <tabstop>widthValue</tabstop>
  <tabstop>latValue</tabstop>
  <tabstop>longValue</tabstop>
  <tabstop>durationHelp</tabstop>
  <tabstop>carryHelp</tabstop>
  <tabstop>rangeHelp</tabstop>
  <tabstop>huntHelp</tabstop>
  <tabstop>killHelp</tabstop>
  <tabstop>growthHelp</tabstop>
  <tabstop>dimensionsHelp</tabstop>
  <tabstop>geoHelp</tabstop>
  <tabstop>advancedOptions</tabstop>
  <tabstop>submit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
